pkg/subctl/cmd/show_connections.go

```diff
@@ -72,14 +63,19 @@ func showConnections(cmd *cobra.Command, args []string) {
 	for _, item := range configs {
 		fmt.Println()
 		fmt.Printf("Showing information for cluster %q:\n", item.clusterName)
-		status := getConnectionsStatus(item.config)
-		printConnections(status)
+		showConnectionsFor(GetSubmarinerResource(item.config))
 	}
 }
 
 func showConnectionsFromConfig(config *rest.Config) {
-	status := getConnectionsStatus(config)
-	printConnections(status)
+	showConnectionsFor(GetSubmarinerResource(config))
+}
```

Comment:
I don't think we need this function anymore. Same with similar functions for endpoints et al.

Reply:
This is called in show_all.go. See comment https://github.com/submariner-io/submariner-operator/pull/1017#discussion_r553229721

Reply:
Right but that should call `showConnectionsFor` instead as Mike pointed out which renders `showConnectionsFromConfig` unused/unnecessary.

Reply:
All these `show*FromConfig` functions are only called once from `showAll` and not being used elsewhere.
As they're just one line functions and don't add any value, I made suggestions to replace them with proper calls.